docs/src/main/paradox/kubernetes-deployment/building-using-sbt.md

```diff
@@ -0,0 +1,82 @@
+# Building using SBT
+
+sbt uses a plugin called [sbt-native-packager](https://www.scala-sbt.org/sbt-native-packager/) to allow conveniently packaging 
+Java and Scala applications built using sbt as Docker images.
+
+
+## Setup
+
+To use this plugin in your sbt application, add the following to your `project/plugins.sbt` file:
+
+@@@vars
+```scala
+addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "<latest.version>")
+```
+@@@
+
+Now you can enable the appropriate plugins in your build, by modifying your project in `build.sbt`:
+
+```scala
+enablePlugins(JavaAppPackaging, DockerPlugin)
+```
+
+Here we're telling native packager to package our application as a Java application that can be run from the command line. This will package up all the applications dependencies (jar files), and generate a start script to start the application. To generate this start script, native packager needs to know what the applications main class is. When the application only has one main class in its source folder, sbt will detect this automatically, but in case there are multiple, or the main class comes from a dependency, it can be set in `build.sbt` like so:
+
+```scala
+mainClass in Compile := Some("akka.sample.cluster.kubernetes.DemoApp")
+```
+
+### Selecting a JDK
+
+By default, sbt native packager uses the `openjdk` latest Docker image from DockerHub. At time of writing, this will give you OpenJDK 11, which is not certified by Lightbend, and it also gives you the Debian OpenJDK build, 
+which is not certified by Lightbend either. For a full list of Lightbend certified JDK builds and versions, see [here](https://developer.lightbend.com/docs/reactive-platform/2.0/supported-java-versions/index.html).
+
+We recommend using the AdoptOpenJDK base image:
+
+```scala
+dockerBaseImage := "adoptopenjdk/openjdk8"
```

Comment:
can we recommend a jdk11 instead?

Reply:
Perhaps let's do that in a separate PR to let others chime in? I'd like to switch us to using a JRE  and 11.